README.md

```diff
@@ -37,14 +37,16 @@ Python 3 is required.
 #### Python
 With `pip` installed, run the following in the root directory:
 
-    pip install -e
+    pip install -e .
 
 To avoid any conflicts with local packages, we recommend using a virtual
 environment. In the root directory:
 
     virtualenv venv
     source venv/bin/activate
 
+Alternatively, if you are using python to manage virtual environments using the `venv` module, then the first line above can be replaced by `python -m venv venv` (where the second `venv` is the virtual environment name).
+
 See [virtualenv docs](https://virtualenv.pypa.io/en/latest/) for more details.
```

Comment:
This line could be moved above the sentence before, but not critical

Reply:
good point!